### PR TITLE
Payment Apps

### PR DESCRIPTION
**Pull request by adrianhopebailie**
- Added new editors 
- Major cleanup to proposal
- Removed normative ref to manifest

### CODE REVIEW

File: proposals/paymentapps/payment-apps.html [hunk 1]
```diff
@@ -1,64 +1,78 @@
...
+                },
+                {   name:       "Tommy Thorsten",
+                    url:        "",
+                    company:    "Opera",
+                    companyURL: "https://opera.com"
+                },
+                {   name:       "Adam Roach",
+                    url:        "",
+                    company:    "Mozilla",
+                    companyURL: "https://mozilla.com"
```
Comment (adamroach) [inline proposals/paymentapps/payment-apps.html]: mozilla.org, please. :)

Comment (adrianhopebailie) [inline proposals/paymentapps/payment-apps.html]: Fixed! Sorry

File: proposals/paymentapps/payment-apps.html [hunk 2]
```diff
@@ -1,64 +1,78 @@
...
+            edDraftURI:   "https://w3c.github.io/webpayments/proposals/paymentapps/payment-apps.html",
             specStatus: "unofficial",
             editors: [
                 {   name:       "Adrian Hope-Bailie",
-                    company:    "Ripple" },
+                    url:        "https://github.com/adrianhopebailie",
+                    company:    "Ripple",
+                    companyURL: "https://ripple.com"
+                },
+                {   name:       "Tommy Thorsten",
```
Comment (tommythorsen) [inline proposals/paymentapps/payment-apps.html]: "Tommy Thorsen", please :)

Comment (adrianhopebailie) [inline proposals/paymentapps/payment-apps.html]: Oh man! Fixed

**Comment by ianbjacobs** [comment]
Hi Adrian,
I don't mind merging the PR. I have a number of comments on the specification that I will make following the PR merger.

Ian

**Comment by adrianhopebailie** [comment]
@ianbjacobs - thanks, also have some more PRs to submit addressing the comments that have been made to date

**Comment by jnormore** [comment]
:+1: I'm good to merge this as-is and iterate.

**Pull request merged**